packages/ckeditor5-table/src/tablecolumnresize/tablecolumnresizeediting.js

```diff
@@ -401,35 +389,35 @@ export default class TableColumnResizeEditing extends Plugin {
 
 		// At this point we change only the editor view - we don't want other users to see our changes yet,
 		// so we can't apply them in the model.
-		editingView.change( writer => this._applyResizingAttributesToTable( writer, target, viewTable ) );
+		editingView.change( writer => this._applyResizingAttributesToTable( writer, viewTable ) );
 	}
 
 	/**
-	 * Calculate the dom column widths. It is done by taking the width of the widest cell
-	 * from each table column (we rely on the TableWalker to determine
-	 * which column the cell belongs to).
+	 * Calculates the dom column widths. It is done by taking the width of the widest cell
+	 * from each table column (we rely on the  {@link module:table/tablewalker~TableWalker}
+	 * to determine which column the cell belongs to).
 	 *
 	 * @private
 	 * @param {module:engine/model/element~Element} modelTable A table which columns should be measured.
-	 * @returns {Array.<Number>} Widths expressed in pixels (without unit).
+	 * @returns {Array.<Number>} Column widths expressed in pixels (without unit).
 	 */
 	_calculateDomColumnWidths( modelTable ) {
```

Comment:
I think we should take the chance and extract `_calculateDomColumnWidths()` method as a local helper in `_onMouseDownHandler()` as this is the only place where it is used.

Reply:
Let's keep the JSDocs though by changing them to `// ` style comments. This API docs might be useful for some in the future.